### PR TITLE
updated pdfelement (5.5.0)

### PR DESCRIPTION
**Pull request by thehunmonkgroup**
#### Editing an existing cask
- [x] Commit message includes cask’s name (and new version, if applicable).
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.

### CODE REVIEW

File: Casks/pdfelement.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 cask 'pdfelement' do
-  version '5.3.9'
-  sha256 'a958f592125f6b518fd9b3d69fa430d58937d2bbeab3efd1eb029348f140c5eb'
+  version '5.5.0'
+  sha256 '319091fe7faad5f203735702c9d97fbbabfa74dd83dfd768ff97cfe11f0ff341'
 
-  url 'http://download.wondershare.net/mac-pdfelement_full973.dmg'
+  url 'http://download.wondershare.com/mac-pdfelement_full841.dmg'
```
Comment (vitorgalvao) [inline Casks/pdfelement.rb]: Where are you getting this from? I still see the old link on the site.

Comment (thehunmonkgroup) [inline Casks/pdfelement.rb]: https://pdf.wondershare.com/, 'Try now' button. I've verified it installs version 5.5.0

Comment (vitorgalvao) [inline Casks/pdfelement.rb]: That’s a different homepage than the one used on this cask, though, so which one is correct?

Comment (thehunmonkgroup) [inline Casks/pdfelement.rb]: Yeah, apparently they have multiple ways to get to the product -- confusing...

This is the URL listed in the Cask currently: http://www.wondershare.net/mac-pdfelement

The 'Try it Free' button on that page links to the download listed in the updated Cask file. Although the version on their homepage says 5.4, the software itself claims it's 5.5.0, so I went with that.

Comment (vitorgalvao) [inline Casks/pdfelement.rb]: Please change the homepage, then.

Comment (thehunmonkgroup) [inline Casks/pdfelement.rb]: So changed

**Pull request merged**